exercises/concept/calculator-conundrum/.meta/src/reference/java/CalculatorConundrum.java

```diff
@@ -2,30 +2,27 @@
 
     public String calculate(int operand1, int operand2, String operation) {
         int result;
-        try {
-            if (operation == null) {
-                throw new IllegalArgumentException("Operation cannot be null");
-            }
-            switch (operation) {
-                case "+":
-                    result = SimpleOperation.addition(operand1, operand2);
-                    break;
-                case "*":
-                    result = SimpleOperation.multiplication(operand1, operand2);
-                    break;
-                case "/":
+        if (operation == null) {
+            throw new IllegalOperationException("Operation cannot be null", new IllegalArgumentException());
```

Comment:
You do not need to have a cause exception. Just have two different constructors for IllegalOperationException: one that takes a message and another that takes a message and a cause.